src/parameters.toml

```diff
@@ -186,6 +186,60 @@ value = 0.96
 type = "float"
 description = "Emissivity of soil, approximated as context. Reference: CLM5 Documentation, 2020. "
 
+[soil_momentum_roughness_length]
+value = 0.01
+type = "float"
+description = "Source: CLM Tech note 5"
+
+[soil_scalar_roughness_length]
+value = 0.007
+type = "float"
+description = "Source: CLM Tech note 5, using u* of 5m/s"
+
+[snow_momentum_roughness_length]
+value = 0.0024
+type = "float"
+description = "Source: CLM Tech note 5"
+
+[snow_scalar_roughness_length]
+value = 0.08
+type = "float"
+description = "Source: CLM Tech note 5, using u* of 5m/s"
+
+[particle_density_quartz]
+value = 2.66e3
+type = "float"
+
+[particle_density_minerals]
+value = 2.65e3
+type = "float"
+
+[particle_density_organic_matter]
+value = 1.3e3
+type = "float"
+
+[vol_heat_capacity_quartz]
+value = 2.01e6
+type = "float"
+
+[vol_heat_capacity_organic_matter]
+value = 2.51e6
+type = "float"
+
+[vol_heat_capacity_minerals]
+value = 2.01e6
+type = "float"
+
+[soil_photosynthetically_active_albedo]
```

Comment:
Im not 100% sure if these should go in...I think they depend on soil type and should be read in from maps.

Reply:
could we wait on this?